### PR TITLE
G search

### PR DESCRIPTION
**Pull request by gmartinez948**
Adds dropdown to nav bar and search page

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -2,38 +2,39 @@ import { useState } from "react";
...
-      main: '#c62828'
+      main: "#c62828",
     },
     warning: {
-      main: '#801f12'
-    }
-  }
+      main: "#801f12",
+    },
+  },
```
Comment (arunbhardwaj) [inline pages/_app.js:37]: nice use of themes

**Review by arunbhardwaj — COMMENTED** [review]
Overall pretty damn fine 👍 do go over the comments I left, they may deem useful.

File: components/Dropdown/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          labelId="demo-controlled-open-select-label"
+          id="demo-controlled-open-select"
+          open={open}
+          onClose={handleClose}
+          onOpen={handleOpen}
+          value={selectedCategory}
+          label="category"
+          onChange={handleChange}
+          variant="outlined"
+        >
```
[the remark this answers is not in the document]
Comment (gmartinez948) [reply components/Dropdown/index.js:49]: onOpen and onClose are functions that material UI <Select> takes in to handle the opening and closing of a dropdown.. For more clarity, travel to: https://mui.com/api/select/ 

The details are under the props section.

File: components/Header.js [hunk 3]
```diff
@@ -20,13 +21,16 @@ const Header = () => {
         // justifycontent: "space-evenly",
         // alignitems: "baseline",
         alignSelf: "auto",
-        color: '#c7d6d5'
+        color: "#c7d6d5",
       }}
```
[the remark this answers is not in the document]
Comment (gmartinez948) [reply components/Header.js:25]: For sure, this is kelly's code. I think I merged it with my branch first and then pushed up.

File: components/ItemCard/CardImage.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { CardMedia } from "@mui/material";
+
+const CardImage = () => {
+  // going to have to play with the image height and width to fit each photo
```
[the remark this answers is not in the document]
Comment (gmartinez948) [reply components/ItemCard/CardImage.js:4]: I'll keep this in mind.

File: components/ItemCard/index.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import { useState } from "react";
+
+import CardImage from "./CardImage.js";
+import CardDescription from "./CardDescription.js";
+import UsernameButton from "./UsernameButton.js";
+
+const ItemCard = () => {
+  const [isFavClicked, setFavClicked] = useState(false);
```
[the remark this answers is not in the document]
Comment (gmartinez948) [reply components/ItemCard/index.js:11]: Ok, going to change that right now. That's helpful, thanks!

File: components/VirtualizedList/index.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  ListItemButton,
+  ListItemText,
+  Box,
+  Typography,
+} from "@mui/material";
+
+// LOOK INTO FIXEDSIZELIST AND HOW IT CAN WORK WITH ARRAYS
+
+const List = () => {
+  const categories = ["Yeezy", "KSubi", "Mike", "Kelly", "Kyle"];
```
[the remark this answers is not in the document]
Comment (gmartinez948) [reply components/VirtualizedList/index.js:13]: This file isnt being used. This is from when I was exploring with the side bar. I thought to leave it there just incase we wanted to implement a side bar with a list

File: features/Search/SearchPage.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+                  md={6}
+                  lg={3}
+                  style={{ background: "inherit" }}
+                >
+                  <ItemCard />
+                </Grid>
+              );
+            })}
+          </Grid>
+        </Box>
```
[the remark this answers is not in the document]
Comment (gmartinez948) [reply features/Search/SearchPage.js:61]: Yes, <Box></Box> is the same as a div.. it comes with material ui

File: pages/_app.js (hunk 1 above)
Comment (gmartinez948) [reply pages/_app.js:37]: That's all Kelly!

File: components/Dropdown/index.js [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  Box,
+} from "@mui/material";
+
+import { useState } from "react";
+
+const Dropdown = () => {
+  const [selectedCategory, setSelectedCategory] = useState("");
+  const [open, setOpen] = useState(false);
+
+  const categories = ["Yeezy", "KSubi", "Mike", "Kelly", "Kyle"];
```
[the remark this answers is not in the document]
Comment (gmartinez948) [reply components/Dropdown/index.js:16]: You're the best category Arun.. they must be logged in to see you lol. And word, Im going to look into enums

File: components/VirtualizedList/index.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (arunbhardwaj) [reply components/VirtualizedList/index.js:13]: OO okay, try adding a `@deprecated` JSdoc flag to the top of the component declaration, something like this

```
/**
* @deprecated
* ...
*/

**Pull request merged**